src/util/Stage.java

```diff
@@ -0,0 +1,48 @@
+package util;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class Stage
+{
+    private Stage instance;
+    private JButton Start;
+
+    private void Stage()
+    {
+
+    }
+
+    public void getInstance()
```

Comment:
This method does not fulfill its purpose since it only creates a Stage object if instance is null, but instance is never upated and nothing is never returned. Please adjust the code (see e. g. the [InputManager Singleton](https://github.com/Cooperr06/q11-doodle-jump-project/blob/main/src/manager/InputManager.java#L8-L21))